docs/tutorial.md

```diff
@@ -394,7 +394,87 @@ What's going on here? Let's step through the code.
 
 (sec_extracting_individuals)=
 
-## Extracting particular SLiM individuals
+## Extracting SLiM individuals
+
+### Extracting a sample of individuals
+
+Often we would like to sample a number of individuals from a population.
+This section provides an example workflow and draws
+attention to some common errors.
+
+The first, most common method to extract individuals is simply to get all
+those that were alive at a particular time,
+using {meth}`.SlimTreeSequence.individuals_alive_at`. For instance, to get
+the list of individual IDs of all those alive at the end of the
+simulation (i.e., zero time units ago), we could do:
+
+```{code-cell}
+orig_ts = pyslim.load("example_sim.trees")
+alive = orig_ts.individuals_alive_at(0)
+
+print(f"There are {len(alive)} individuals alive in the final generation.")
+```
+
+These are individual IDs. One way to take a sample of living individuals
+and write their SNPs to a VCF is:
+
+```{code-cell}
+np.random.seed(1)
+keep_indivs = np.random.choice(alive, 100, replace=False)
+ts = pyslim.SlimTreeSequence(msprime.mutate(orig_ts, rate=1e-8, random_seed=1))
+with open("example_snps.vcf", "w") as vcffile:
+    ts.write_vcf(vcffile, individuals=keep_indivs)
+```
+
+
+### Extracting individuals after simplification
+
+In cases where we want to obtain genotypes *after* simplification, we
+need to pay attention to specific node IDs. As
+mentioned earlier, {meth}`.SlimTreeSequence.simplify` takes a list
+of nodes as input:
+
+```{code-cell}
+keep_nodes = []
+for i in keep_indivs:
+    keep_nodes.extend(orig_ts.individual(i).nodes)
+sts = rts.simplify(keep_nodes)
+```
+
+There are often some nodes retained
```

Comment:
there are (almost) *always* nodes retained that are not samples. Do you mean that there are often non-sample *individuals*?